typescript/web/src/components/workspace-switcher/create-workspace-modal/create-workspace-button.tsx

```diff
@@ -0,0 +1,18 @@
+import { Button } from "@chakra-ui/react";
+import { FC } from "react";
+
+export const CreateWorkspaceButton: FC<{
+  isDisabled?: boolean;
+  isLoading?: boolean;
+}> = ({ isDisabled, isLoading }) => (
```

Comment:
- [ ] Using `FC` adds `children` to the props which is not expected in this case.
- [ ] Props should be defined in a dedicated `type CreateButtonProps = Pick<ButtonProps, "isDisabled" | "isLoading">`

Reply:
Fixed
(FYI, I've reused a component you introduced in a previous commit 😉 )

Reply:
Thanks!
Yes, it does not surprise me. I used to type functions components with `FC` on previous projects. But it was a mistake as it's implies that a component can have some children. [More info](https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13)